packages/cli/src/commands/setup/deploy/providers/render.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from 'src/lib'
+
+const RENDER_YAML = `services:
+- type: web
+  name: redwood-api
+  env: node
+  plan: starter
+  buildCommand: yarn && yarn rw prisma migrate deploy && yarn rw build api
+  startCommand: cd api && yarn api-server --functions ./dist/functions
+  envVars:
+  - key: DATABASE_URL
+    fromDatabase:
+      name: redwood-db
+      property: connectionString
+  - key: NODE_VERSION
+    value: 14.16.0
+
+- type: web
+  name: redwood-web
+  env: static
+  buildCommand: yarn && yarn rw build web
+  staticPublishPath: ./web/dist
+  envVars:
+  - key: API_PROXY_HOST
+    fromService:
+      name: redwood-api
+      type: web
+      property: host
+  routes:
+  - type: rewrite
+    source: /*
+    destination: /index.html
+
+databases:
+- name: redwood-db
+`
+const RENDER_HEALTH_CHECK = `
+export const handler = async () => {
+  return {
+    statusCode: 200,
+  }
+}
+`
+const NODE_VERSION = `14.16.0`
+
+// any packages to install
+export const apiPackages = ['@redwoodjs/api-server']
+
+// any files to create
+export const files = [
+  { path: path.join(getPaths().base, 'render.yaml'), content: RENDER_YAML },
+  {
+    path: path.join(getPaths().base, 'api/src/functions/healthz.js'),
+    content: RENDER_HEALTH_CHECK,
+  },
+  { path: path.join(getPaths().base, '.node-version'), content: NODE_VERSION },
+]
+
+// any edits to Prisma data source
+export const prismaDataSourceEdit = () => {
```

Comment:
Ah, this raises an interesting question about how to handle potential DB Schema conflicts. I don't believe we should handle it this way because it could break local development (e.g. user has SQLite in use locally for development and is "checking out Render deploy setup", which I recognize is an edge case).

I suggest a better way to handle this would be:
1. keep the check provider =  postres?
2. If false, output a WARNING message with instructions to change DB provider in prisma.schema

Reply:
outdated